### DISCUSSION TITLE
Right clicking on mods should open the mod customization panel

### DISCUSSION THREAD

**Discussion by Arnacino** [ppy/osu#21624] [Ideas] [answered]
As the title says, I think right clicking on mods should open the mod customization panel in some way. 
image below for clearing out
![immagine](https://user-images.githubusercontent.com/47191151/207150157-7c6d716e-f0de-4d53-978a-ccaf4720b47a.png)

**Comment by mk56-spn** [comment, marked as answer]
Sounds nice to have. Happy to take this one up myself if it's given the go ahead

**Reply by peppy** [reply]
Sounds fine as long as it still selects as well.

**Reply by bdach** [reply]
This has been suggested in the past and I think I had this ready on a branch even but then promptly lost it somewhere... 

You can try if you wish but I will likely be very particular about the implementation. It should be using `ModState.PendingConfiguration`. If you do it right this task should take 15 minutes tops. (Not counting tests.)

**Reply by mk56-spn** [reply]
thanks for the advice, will do my best to conform to expectations
